framework/DataObjects/HistorySet.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
How is the HistorySet used in the rework branch?  Shouldn't it be the XHistorySet?

Reply:
this is caused by the merging of devel on the top of this branch.